<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentPage</class>
 <widget class="QWidget" name="PaymentPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>911</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PaymentPage</string>
  </property>
  <widget class="QPushButton" name="btnAlipay">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>350</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>249</width>
     <height>12</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image:url(:/new/prefix1/images/alipy.png);
border-radius: 20px;   
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBankCard">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image:url(:/new/prefix1/images/visa.png);
border-radius: 20px;   
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWeChat">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image:url(:/new/prefix1/images/wechatpy.png);
 border-radius: 20px;    /* 设置边框圆角为50像素，以创建椭圆形状 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
